liuliu/update-webhook-doc (#1507)
dolthub/docs: content/products/dolthub/dolthub-api/hooks.md

The pull request event webhook is triggered whenever a pull request is created, merged, or closed. The payload includes these fields:

| Key                            | Type   | Description                                                                                       |
| ------------------------------ | ------ | ------------------------------------------------------------------------------------------------- |
| `event_name`                   | string | The name of the event triggered by the webhook. In this case, it will always be "pull_request".   |

> | `repository.name`              | string | The name of the database where the pull request is located.                                       |

Reviewer: ```suggestion
| `repository.name`              | string | The name of the database for the pull request base.                                       |
```

> | `repository.owner`             | string | The owner (username or org name) of the database where the pull request is located.               |

Reviewer: ```suggestion
| `repository.owner`             | string | The owner (username or org name) of the pull request base.               |
```

| `pullID`                       | string | The ID of the pull request that triggered the webhook.                                            |
| `action`                       | string | The action that triggered the webhook (Opened, Closed, Merged.).                                  |
| `state`                        | string | The current state of the pull request after the event.                                            |
| `fromRepository.name`          | string | The name of the database where the pull request is created from.                                  |
| `fromRepository.name`          | string | The owner of the database where the pull request is created from.                                 |
| `author`                       | string | The author of the pull request.                                                                   |
| `sender`                       | string | The user who triggered the event.                                                                 |

An example of the payload object:

```json
{
  "event_name": "pull_request",
  "repository": {
    "name": "test",
    "owner": "dolthub"
  },
  "pullID": "15",

>   "action": "Opened",

Reviewer: add `state` here too.
